python/mlc_chat/compiler/quantization/utils.py

```diff
@@ -18,17 +18,11 @@ def convert_uint_to_float(  # pylint: disable=too-many-arguments
         shape=[weight.shape[0], weight.shape[1] * num_elem_per_storage]
         if out_shape is None
         else out_shape,
-        fcompute=lambda i, j: tir.Cast(
```

Comment:
@LeshengJin @cyx-6 let's always avoid using TIR's IR node constructor, e.g. tir.Cast, tir.Add, but instead using TIR's convenient functions, e.g. tir.cast, tir.add, which does constant folding and more friendly type promotion, etc